cpp/src/parquet/properties.h

```diff
@@ -192,6 +197,8 @@ class PARQUET_EXPORT ColumnProperties {
 
   int compression_level() const { return compression_level_; }
```

Comment:
Since `Codec` contains `compression_level_`, would this be duplicated or useless?

Reply:
Now we have a `compression_level` and a `CodecOptions`. Previously, user can set `compression_level`, and the compression level would be changed.

After this patch, if user change the compression level, the codec will not be changed.

Can we make `CodecOptions` mutate it's `compression_level`, and make `set_compression_level` set the level in `CodecOptions` directly?

Reply:
that makes sense :)

Reply:
Can we just get it from `codec_options_`?

Reply:
+1 on this. `compression_level_` is duplicated.